CHANGELOG.md

```diff
@@ -0,0 +1,65 @@
+
+# 1.0.0
+
+The `Carbon.Windows.HttpServer` module is a port of the following functions from the `Carbon` module:
+
+* Get-CSslCertificateBinding
+* Remove-CSslCertificateBinding
+* Set-CSslCertificateBinding
+* Test-CSslCertificateBinding
+
+## Upgrade Instructions
+
+All functions now require the `C` prefix:
+
+* Rename usages of `Get-SslCertificateBinding` to `Set-CSslCertificateBinding`.
+* Rename usages of `Remove-SslCertificateBinding` to `Remove-CSslCertificateBinding`.
+* Rename usages of `Set-SslCertificateBinding` to `Set-CSslCertificateBinding`.
+* Rename usages of `Test-SslCertificateBinding` to `Test-CSslCertificateBinding`.
+
+Update usages of `Get-SslCertificateBindings` to `Get-CSslCertificateBinding`. We removed the
+`Get-SslCertificateBindings` alias.
+
+Update usages of `Get-CSslCertificateBinding` that use the `IPAddress` or `Port` parameter with `-ErrorAction Ignore`.
+The `Get-CSslCertificateBinding` function now writes an error if searching for a binding and it doesn't exist.
+
+Update usages of `Remove-CSslCertificateBinding` with `-IPAddress '0.0.0.0'` and `-Port 443`. The function no longer
+deletes those bindings by default.

```

Comment:
This is confusing, maybe reword to something like:

> Update usages of Get-CSslCertificateBinding to include parameters -IpAddress '0.0.0.0' and -Port 443. The function no longer deletes those binding by default.

Reply:
Updates wording. Also changed to allow deleting all bindings, but requests confirmation from the user first.